docs/dev/build-dependencies.md

```diff
@@ -0,0 +1,16 @@
+# Build Dependency Installation
```

Comment:
Probably squash this content into a new section of the new-in-#328 `docs/dev/dependencies.md` (and thanks for hanging in there despite the flux :).

Reply:
done!